{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -13,7 +13,10 @@ max-line-length = 120
 ; D200 = One-line docstring should fit on one line with quotes
 ; D203 = 1 blank line required before class docstring
 ; D212 = Multi-line docstring summary should start at the first line
-ignore = D101,D200,D203,D212
+; D406 = Section name should end with a newline (numpy style, clashes with Google style)
+; D407 = Missing dashed underline after section (numpy style again)
+; D413 = Missing blank line after last section (numpy style again)
+ignore = D101,D200,D203,D212,D406,D407,D413
```

Comment:
http://www.pydocstyle.org/en/latest/error_codes.html#default-conventions

> The pep257 convention, which is enabled by default in pydocstyle, checks for all of the above errors except for D203, D212, D213, D214, D215, D404, D405, D406, D407, D408, D409, D410, and D411 (as specified in PEP257).

Can you help me understand why we aren't just ignoring this whole list?